councilmatic_core/models.py

```diff
@@ -6,8 +6,9 @@
 import inspect
 import importlib
 
-if not hasattr(settings, 'OCD_CITY_COUNCIL_ID'):
-    raise ImproperlyConfigured('You must define a OCD_COUNCIL_ID in settings.py')
+if not hasattr(settings, 'OCD_CITY_COUNCIL_ID') or not hasattr(settings, 'OCD_CITY_COUNCIL_NAME'):
```

Comment:
this conditional should be `if not (has_id or has_name)` or `if not has_id and not has_name`